### PR TITLE
started linking create item with backend

### PR DESCRIPTION
**Pull request by SiddharthKartha**
Dynamic list view for the item descriptors. Added async call for submit button. Can't complete implementation until the Inventory table is created.

### CODE REVIEW

File: frontend/src/components/createitem.component.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    let response;
+    try {
+      response = await axios.post("http://localhost:5000/inventory", inventory);
+    } catch (error) {
+      console.log(error);
+    }
+    response.then((result) => {
+      /*let item = result["data"]["item"];*/
+      if (result !== undefined && result.status === 201);
+      /*setInventories([...inventories, inventory]);*/
```
Comment (ujwal-jain) [inline frontend/src/components/createitem.component.js:17]: Delete this since it's not needed

**Comment by JihoPark21** [comment]
Looks good to me

File: frontend/src/components/createitem.component.js (hunk 1 above)
Comment (SiddharthKartha) [reply frontend/src/components/createitem.component.js:17]: done on local file. Will commit and push along with next batch of changes.

**Comment by harsimran-preet** [comment]
Getting better one step at a time. Great work.

File: frontend/src/components/createitem.component.js (hunk 1 above)
Comment (ujwal-jain) [reply frontend/src/components/createitem.component.js:17]: Cool you should resolve conflicts also

**Pull request closed without merging**